packages/components/src/presentation-components/table/UsePresentationTable.ts

```diff
@@ -61,8 +61,8 @@ export interface UsePresentationTableResult<TColumns, TRow> {
 export interface UsePresentationTableWithUnifiedSelectionResult<TColumns, TRow> extends UsePresentationTableResult<TColumns, TRow> {
   /** Specifies rows that have been selected (toggled) by other components on the appropriate selection level. */
   selectedRows: TRow[];
-  /** Callback to use when a row is selected. */
-  onSelect: (selectedData: string[]) => void;
+  /** Callback to use when a row is selected. It needs to be called with keys of selected rows that have been converted to strings. */
+  onSelect: (selectedKeys: string[]) => void;
```

Comment:
I don't think API consumers need to know anything about keys being converted to strings - that's an implementation detail.

```suggestion
  /** 
   * A function that should be called when a table row is selected.
   * @param selectedRowKeys Keys of selected table rows. These should match `TableRowDefinition.key` passed to `UsePresentationTableProps.rowMapper` function when new rows are loaded.
   */
  onSelect: (selectedRowKeys: string[]) => void;
```